lib/stream.js

```diff
@@ -38,6 +39,8 @@ Stream.PassThrough = require('_stream_passthrough');
 Stream.pipeline = pipeline;
 Stream.finished = eos;
 
+Stream.promises = promises;
```

Comment:
We probably want to lazy-load this to not slow down programs - Look at how fs/promises is lazy loaded and emulate the same pattern?

Reply:
the promises.js already lazy loads, not sure how many levels of lazy loading makes sense...

Reply:
Where?

Reply:
`internal/streams/promises.js` has lazily loaded the required modules inside. We can also add another layer of lazy loading here.

Reply:
fixed